types/src/ledger/log.rs

```diff
@@ -121,6 +123,28 @@ impl VerifiableLog {
             })
     }
 
+    fn sort_attestations_by_timestamp(&self) -> Vec<&TimestampedHeadlessAttestation> {
+        self.pod_metadata
+            .attestations
+            .iter()
+            .sorted_by_key(|a| a.timestamp.as_seconds())
```

Comment:
`Timestamp::as_seconds()` involves dividing microseconds by 1M. How about comparing micros or implementing `PartialOrd` for `Timestamp`?